### ISSUE TITLE
Option for double qoutes for Alphanumeric in CSV

### ISSUE THREAD

**Issue by samalik123** [benkeen/generatedata#280]
It will be great to have an option for CSV file generation to export with double quotes for Alphanumeric values.

Thanks

**Comment by benkeen** [comment]
Yes, this would be useful. Maybe just a blanket "[x] enclose all data in double quotes" option. I think right now it's clever and only adds it when it thinks it's necessary.
